lib/canonical_version.sh

```diff
@@ -1,4 +1,28 @@
-canonicalVersion() {
+#!/usr/bin/env bash
+
+fetch_erlang_versions() {
+  url="https://raw.githubusercontent.com/HashNuke/heroku-buildpack-elixir-otp-builds/master/otp-versions"
+  curl -s "$url" 
+}
+
+exact_erlang_version_available() {
+  version=$1
+  available_versions=$2
+  found=1
+  while read -r line; do
+    if [ "$line" = "$version" ]; then
+      found=0
+    fi
+  done <<< "$available_versions"
+  echo $found
```

Comment:
what's the point of echoing `1` here? probably should be `return 1`

Reply:
Same as above